src/components/analyticVisualization/VisualizationSearch/helper.js

```diff
@@ -97,3 +98,6 @@ const isValidOrgUnit = visualization =>
     visualization.userOrganisationUnit ||
     visualization.userOrganisationUnitChildren ||
     visualization.userOrganisationUnitGrandChildren
+
+export const orderVisualizations = visualizationList =>
+    orderBy(visualizationList, [item => item.valid === true], ['desc'])
```

Comment:
I was going to suggest `orderBy(visualizationList, ['valid'], ['desc'])` but it looks like there are a lot of visualizations that have null value for valid (I don't know the app logic well enough to comment if that is appropriate or not.